src/components/atoms/Button/v2/Button.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { Box, styled, Typography } from '@mui/material'
+import colors from '@assets/scss/variables/_color.variables.module.scss'
+
+type ButtonBoxParameters = {
+  width?: string
+  disabled?: boolean
+  fontColor?: string
+  backgroundColor?: string
```

Comment:
I would like to use the type `Colors` instead of string if the idea is to restrict the colors to our own set of colors. If that's not the idea, maybe use `Colors | string`, so If we want to use our set of colors, we don't need to import the CSS file.

Example: As we have on BadgePreviewV2 or Stepper

Reply:
I think is much better to provide the colors list (same with gradients and other variables) in the index file so its easy accessible for the user of the library and handle the colors as strings. Using the type Color you have to specify the color in a string anyway and cast it, and then you don't have autocomplete or you can not see easy all available colors. The type Color is only better for storybook but not at the moment of usage of a component.